### PR TITLE
lesson02+lesson03 tasks

### PR DESCRIPTION
**Pull request by jemiekam**
@kamwar

### CODE REVIEW

**Comment by jemiekam** [comment]
@kamwar added lesson03 tasks

File: students/jemielity_kamil/lesson_02_flow_control/chocolate_bar.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+n = int(input("Length of chocolate: "))
+m = int(input("Width of chocolate "))
+k = int(input("Number of squares: "))
+
+if k > n*m:
```
Comment (kamwar) [inline students/jemielity_kamil/lesson_02_flow_control/chocolate_bar.py]: It's not stated in PEP8 but it is good practice to use space between math operators 'if k > n * m:'

File: students/jemielity_kamil/lesson_02_flow_control/knight_move.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+ending_column = int(input('Ending column: '))
+ending_row = int(input('Ending row: '))
+
+if abs(starting_column - ending_column) == 2:
+    if ending_row == starting_row + 1 or starting_row - 1:
+        print('YES')
+    else:
+        print('NO')
+elif abs(starting_row - ending_row) == 2:
```
Comment (kamwar) [inline students/jemielity_kamil/lesson_02_flow_control/knight_move.py]: My first impression is that 'elif abs(starting_row - ending_row) == 2:' should be merged with 'if abs(starting_column - ending_column) == 2:'

I noticed that this code doesn't pass snakify test. Are you using Snakify tests to verify your solution? If not, please do

File: students/jemielity_kamil/lesson_02_flow_control/ladder.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+number = int(input('Integer: '))
+
+for row in range(1, number+1):
+    for i in range(1, row+1):
+        print(i, end='')
+
+    print(sep='')
```
Comment (kamwar) [inline students/jemielity_kamil/lesson_02_flow_control/ladder.py]: Isn't this print just for an empty line? If yes then it enough to print empty string print('')

File: students/jemielity_kamil/lesson_03_functions/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+### Lesson 3 - Functions
```
Comment (kamwar) [inline students/jemielity_kamil/lesson_03_functions/README.md]: Please always create a new PR for a new lesson

**Pull request closed without merging**